internal/provisioner/plain/main.go

```diff
@@ -165,15 +166,31 @@ func main() {
 		os.Exit(1)
 	}
 
+	const plainBundleProvisionerName = "plain"
+	unpacker := source.NewUnpacker(map[rukpakv1alpha1.SourceType]source.Unpacker{
+		rukpakv1alpha1.SourceTypeImage: &source.Image{
+			Client:          mgr.GetClient(),
+			KubeClient:      kubeClient,
+			ProvisionerName: plainBundleProvisionerName,
+			PodNamespace:    ns,
+			UnpackImage:     unpackImage,
+		},
+		rukpakv1alpha1.SourceTypeGit: &source.Git{
+			Client:          mgr.GetClient(),
+			KubeClient:      kubeClient,
+			ProvisionerName: plainBundleProvisionerName,
+			PodNamespace:    ns,
+			UnpackImage:     unpackImage,
+			GitClientImage:  gitClientImage,
+		},
+	})
```

Comment:
Follow-up: any value in updating the NewUnpacker constructor to work with the options pattern?

Reply:
Hadn't thought too much about that option, but I'm not opposed!